.stylelintrc.yaml

```diff
@@ -0,0 +1,24 @@
+extends: stylelint-config-standard
+plugins:
+  - stylelint-selector-bem-pattern
+  - stylelint-order
+rules:
+  # Override stylelint-config-standard
+  rule-empty-line-before:
+  - always
+  - except:
+    - after-single-line-comment
+    - inside-block-and-after-rule
+    - first-nested
+  # Single-quotes are our convention throughout our codebase within CSS
+  string-quotes: single
+  # Follow best practices
+  font-family-name-quotes: always-where-recommended
+  font-weight-notation: numeric
+  selector-attribute-quotes: always
+  # We use classic BEM syntax as our preferred way to format classes.
+  plugin/selector-bem-pattern:
+    componentName: ^[a-z]+(?:-[a-z]+)*$
+    # <namespace>-<block>__<element>*_<modifier>*[<attribute>]*
+    componentSelectors: ^\.{componentName}(?:__[a-z]+(?:-[a-z]+)*)*(?:_[a-z]+(?:-[a-z]+)*)*(?:\[.+\])*$
+  # Order will be soon
```

Comment:
Do you plan to add order rules in the near future or you're ready to do it in this PR?

Reply:
Right now plugin for bem-pattern isn't working.. and I plan to add properties order

Reply:
@niktariy, so, could you please remove it? Regarding the properties order... my question is, do you plan to add it in this PR?